hydra/utils.py

```diff
@@ -12,38 +13,82 @@
 log = logging.getLogger(__name__)
 
 
+def _safeimport(path: str) -> Optional[ModuleType]:
+    """
+    Import a module; handle errors; return None if the module isn't found.
+    This is a typed simplified version of the `pydoc` function `safeimport`.
+    """
+    from importlib import import_module
+
+    try:
+        module = import_module(path)
+    except ImportError as e:
+        if e.name == path:
+            return None
```

Comment:
what case is this handling?

Reply:
this was a relic from the pydoc method which checks the method from front to back, which is required if one uses `__import__`.  I removed it now.